### DISCUSSION TITLE
Slope Graph in Android Auto

### DISCUSSION THREAD

**Discussion by mrfischer77** [agronick/aa-torque#158] [General]
The Inclinometer, slope works well on mobile, other information also works well on Android Auto.How to put the Slope graphs, like in Torque Pro on mobile?
![Screenshot_2024-10-29-13-09-38-412_org prowl torque](https://github.com/user-attachments/assets/268453df-584f-40a0-ae27-6750f4fc7240)

**Comment by agronick** [comment]
You can use a dial with ray mode enabled which would work good for this.

**Reply by mrfischer77** [reply]
Could you tell more about do it on Android Auto?

**Reply by agronick** [reply]
Is there a PID for that or it it just using the gyroscope in your phone? I'm talking about the first two types of dials here.

![image](https://github.com/user-attachments/assets/b83ac4ab-e036-447e-b71a-afbd2e21e2bd)

**Comment by mrfischer77** [comment]
On the Torque pro, it uses the Cellphone gyroscope.
I think there isn't a PID for this.

It's a special kind of "display" for that.
